agent/src/util/discovery_operator.rs

```diff
@@ -1869,6 +2009,29 @@ pub mod tests {
         // 4: Assert that local devices that go offline are removed from the instance map
         //
         let mut mock = MockKubeInterface::new();
+        let instance: akri_shared::akri::instance::Instance = serde_json::from_str(
```

Comment:
nit: might be worth moving this test instance into a `get_test_instance()` function to reduce lines of code, maybe supporting a configurable nodes list